evalai/challenges.py

```diff
@@ -21,12 +22,19 @@ def challenges(ctx):
 
 @click.group(invoke_without_command=True, name='list')
 @click.pass_context
-def list_challenges(ctx):
+@click.option('-participate', default='false',
+              help="Show the challenges that you've participated")
+@click.option('-host', default='false',
```

Comment:
`--host`, double dash instead of single dash. As default as False instead of 'false' to infer boolean type.